build.gradle

```diff
@@ -33,6 +33,7 @@ allprojects {
         cordaCliBinDir = "${System.getProperty("user.home")}/.corda/cli"
         cpiUploadTimeout = cpiUploadDefault
         cordaProcessorTimeout = processorTimeout
+        vnodeRegistrationTimeout = vnodeRegistrationTimeoutDefault

```

Comment:
should we add a test for this new addition ?

Reply:
We'll be using the regpack for this. I'm looking to add more tests to the reg-pack once https://r3-cev.atlassian.net/browse/CORE-8176 is done (it's another configurable property task)

Reply:
thx for clarifying